compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEUtils.cpp

```diff
@@ -301,6 +302,7 @@ bool isMatmul(linalg::LinalgOp linalgOp) {
 /// its producer is a matmul-like op.
 bool isMatmulProducerOfElementwise(linalg::LinalgOp linalgOp) {
```

Comment:
This function doesn't seem to have a very tight definition (in code or documentation) and it's not tested in a unit test, so I'm wondering why it is is AMDAIEUtils.hpp and not just local to the translation unit it is used in. 

My thinking is that AMDAIEUtils.hpp is for code which might be used in multiple places (or must be in a header for unit testing).

Reply:
This function is already used in multiple places. For another use you could refer to this https://github.com/nod-ai/iree-amd-aie/blob/e296b578f33f04873a94cee000d1d4409d051d9b/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEBufferizeToAllocation.cpp#L50 for example.